tests/test_registercloudguest.py

```diff
@@ -272,7 +272,10 @@ def test_register_cloud_guest_force_reg_zypper_not_running_region_changed(
          SMTregistryName="registry-ec2.susecloud.net"
          region="antarctica-1"/>''')
 
-    smt_server = SMT(etree.fromstring(smt_data_ipv46))
+    child = etree.fromstring(smt_data_ipv46)
+
+    smt_server = SMT(child)
+
```

Comment:
Duh, thanks again for picking up on this. The light for this should have really come on in my head but obviously it didn't. Much appreciate the help.